spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
You're adding this file for a second time. Please review your PR after merging from original master branch.